### PR TITLE
Add "xwl=nowirelink()" and finite/isinf/isnan E2 functions (fixes #1139, #1143)

### LINKED ISSUE #1139: E2: Add "xwl=nowirelink()" function
Add a builtin/core function which would return an invalid wirelink.

Currently this is only possible if "wiring" extension is enabled, like:
```
local NoWirelink = noentity():wirelink()
```

### PR DESCRIPTION
**Pull request by CaptainPRICE**
Fixes:
#1139 
#1143

### CODE REVIEW

File: lua/entities/gmod_wire_expression2/core/custom/cl_wiring.lua [hunk 1]
```diff
@@ -3,4 +3,5 @@ E2Helper.Descriptions["createWire(e:ess)"] = "Creates a wire from specified inpu
 E2Helper.Descriptions["deleteWire(e:s)"] = "Unwires the specified input of the entity"
 E2Helper.Descriptions["getWireInputs(e:)"] = "Returns array of all inputs of the entity"
 E2Helper.Descriptions["getWireOutputs(e:)"] = "Returns array of all outputs of the entity"
-E2Helper.Descriptions["wirelink(e:)"] = "Returns entity's wirelink"
\ No newline at end of file
+E2Helper.Descriptions["nowirelink()"] = "Returns an invalid wirelink"
```
Comment (CaptainPRICE) [inline lua/entities/gmod_wire_expression2/core/custom/cl_wiring.lua]: Whoops, I have added you in the wrong file.

**Comment by CaptainPRICE** [comment]
Damn it, not the master branch... I didn't mean to push number functions into master.

**Comment by Divran** [comment]
Well I was thinking of merging this but it seems you've fucked up your branch lol
I'll just do it manually, whatever

**Comment by oldmud0** [comment]
Good job not giving credit.

**Comment by bigdogmat** [comment]
Is credit really needed for a 3 liner function?

**Comment by oldmud0** [comment]
yeah

**Comment by CaptainPRICE** [comment]
@Divran Oh well, you could have merged this 2-in-1 once you get opinions from others. But, since you closed this 2-in-1, now I have renamed it back. [Here](https://github.com/wiremod/wire/pull/1145) is a separate pull request for number functions that was added from here.

**Pull request closed without merging**